src/sentry/rules/filters/issue_category.py

```diff
@@ -0,0 +1,40 @@
+from collections import OrderedDict
+from typing import Any
+
+from django import forms
+
+from sentry.eventstore.models import Event
+from sentry.rules import EventState
+from sentry.rules.filters import EventFilter
+from sentry.types.issues import GroupCategory
+
+CATEGORY_CHOICES = OrderedDict([(f"{gc.value}", gc.name) for gc in GroupCategory])
```

Comment:
I think we will want to limit category choices based on whether the `organizations:performance-issues` feature is available. If this is overly difficult due to the way these classes are set up we can skip it (maybe wait to merge until GA?)

Reply:
Would adding the feature check on the `ProjectRulesConfigurationEndpoint` be enough? 

We can also wait for GA since I don't think this functionality is crucial.